carbon-design-system/carbon
# fix(tooltip): remove reset mixin from tooltip mixin

Closes #4245 

### Reasoning

Due to the level of specificity of reset styles (like `border: 0`) that can be applied if this mixin is included (see details in #4245)... I think it would be better to include the `reset` mixin directly inside a given component when needed, instead of passing it along via `tooltip--trigger` mixin. 🤔 

### How to test

Need to set `$css--reset: false`.

I tested this for the following components, which appear to be the only places where the `toolitp--trigger` mixin is used (https://github.com/carbon-design-system/carbon/search?q=tooltip--trigger&unscoped_q=tooltip--trigger):
*  `Button` (specifically `hasIconOnly` + tooltip variation)
* `TextInput` (password visibility icon variation)
* `Tooltip`


#### Changelog

**Removed**

- remove `reset` mixin from `tooltip--trigger` mixin
